patches/components-invalidation-impl-push_client_channel.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/components/invalidation/impl/push_client_channel.cc b/components/invalidation/impl/push_client_channel.cc
+index 709d8e562c22..9170819bb45e 100644
+--- a/components/invalidation/impl/push_client_channel.cc
++++ b/components/invalidation/impl/push_client_channel.cc
+@@ -28,6 +28,7 @@ PushClientChannel::PushClientChannel(
+     : push_client_(std::move(push_client)),
+       scheduling_hash_(0),
+       sent_messages_count_(0) {
++  return; // feature disabled in Brave
```

Comment:
I think these patches can also be removed with a chromium_src override on SyncNetworkChannel to subclass `PushClientChannel`